app/lib/common/utilities/address.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
+@immutable
```

Comment:
A nice little keyword that prevents people from accidentally forgetting `final` keyword.